ui-tests/src/test/resources/features/integrations/db-to-db.feature

```diff
@@ -300,3 +300,55 @@ Feature: Integration - DB to DB
 
     Then validate that number of all todos with task "Joe" is "0"
     Then validate that number of all todos with task "Jimmy" is "1"
+
+  @db-insert-multiple-rows
+  Scenario: Inserting multiple rows 
+    Given Set Todo app credentials
+    Then inserts into "todo" table
+      | Joe |
+    Then inserts into "todo" table
+      | Jimmy |
+    When navigate to the "Customizations" page
+    And create new API connector
+        | source   | file          | swagger/connectors/todo.json  |
+        | security | authType      | HTTP Basic Authentication     |
+        | details  | connectorName | TODO-API                      |
+        | details  | host          | http://todo.syndesis.svc:8080 |
+        | details  | baseUrl       | /api                          |
+    And navigate to the "Connections" page
+    And created connections
+        | TODO-API | todo | TODO-app | no validation |
+
+    When navigate to the "Home" page
+    And click on the "Create Integration" button    
```

Comment:
it seems there are some whitespaces at the end of the line, please check that :-)